package.json

```diff
@@ -326,13 +331,14 @@
     "@metamask/message-manager": "^10.1.0",
     "@metamask/message-signing-snap": "^0.4.0",
     "@metamask/metamask-eth-abis": "^3.1.1",
+    "@metamask/multichain": "^1.0.0",
     "@metamask/name-controller": "^8.0.0",
     "@metamask/network-controller": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
     "@metamask/notification-controller": "^6.0.0",
     "@metamask/notification-services-controller": "^0.14.0",
     "@metamask/object-multiplex": "^2.0.0",
     "@metamask/obs-store": "^9.0.0",
-    "@metamask/permission-controller": "^10.0.0",
+    "@metamask/permission-controller": "^11.0.0",
```

Comment:
We should upgrade this in a separate PR to reduce scope. This update is included in this controller update epic: https://github.com/MetaMask/MetaMask-planning/issues/3568
But there is no separate issue for it afaik.

Reply:
https://github.com/MetaMask/metamask-extension/pull/28743